### ISSUE TITLE
assert predicts.shape[0] == labels.shape[0]

### ISSUE THREAD

**Issue by hhjgithub514** [iwtw/pytorch-TP-GAN#7]
Hello. In pretrain.py, predicts.shape[0] is not equal to labels.shape[0]. The value of predicts.shape[0] is four times that of  labels.shape[0]. What does it mean? How can I solve the problem?
